pkgs/by-name/sy/syntax/package.nix

```diff
@@ -18,6 +18,10 @@ buildNpmPackage rec {
 
   npmDepsHash = "sha256-jZwbRGGg4tek6Jr+V7/SceJlsbIv7jFWQ+qa+fnChTw=";
 
+  postInstall = ''
+    find $out/lib/node_modules -xtype l -delete
```

Comment:
I will set `dontCheckForBrokenSymlinks` here instead of removing broken symlinks

https://github.com/NixOS/nixpkgs/pull/461250#discussion_r2525658329